shared/manifest_q1.py

```diff
@@ -0,0 +1,29 @@
+# Q1/Mk4 only files; would not be needed on Mk3 or earlier.
+freeze_as_mpy('', [
+	'psram.py',
+	'mk4.py',
+	'q1.py',
+	'keyboard.py',
+	'scanner.py',
+	'bbqr.py',
+    'decoders.py',
+	'lcd_display.py',
+	'st7788.py',
+	'gpu.py',
+	'vdisk.py',
+	'nfc.py',
+	'ndef.py',
+	'trick_pins.py',
+	'ux_q1.py',
+	'battery.py',
+	'notes.py',
+	'calc.py',
+], opt=0)
```

Comment:
Noticed this is missing the files needed for HSM mode, from reading the other code I don't *think* removing that feature from the Q1 was intentional?

```diff
--- manifest_q1.py
+++ manifest_q1.py
@@ -15,6 +15,9 @@
        'ndef.py',
        'trick_pins.py',
        'ux_q1.py',
+       'hsm.py',
+       'hsm_ux.py',
+       'users.py',
        'battery.py',
        'notes.py',
        'calc.py',
```

**Mk4:**

```python
freeze_as_mpy('', [
	'ssd1306.py',
	'mempad.py',
	'psram.py',
	'mk4.py',
	'vdisk.py',
	'nfc.py',
	'ndef.py',
	'trick_pins.py',
	'ux_mk4.py',
	'hsm.py',
	'hsm_ux.py',
	'users.py',
], opt=0)
```

Reply:
We wanted the flash space for the new larger font and other good stuff. In HSM mode, the Mk4 can do everything the Q can do, since the QR scanner and other improvements are primarily UI that would be wasted in unattended mode.

Reply:
Ah, that makes sense -- is there not enough flash space at all to include it, or would it eat settings space or something? Slgihtly sad but I'll live 😆 

Might be handy to note that it's only Mk4 (and Mk3?) for HSM on the HSM docs https://coldcard.com/docs/hsm/ and Q1 webpage https://coldcard.com/q , I couldn't find a note anywhere other than digging in to the code here (though yes it's a pretty advanced topic so that could be TMI/clutter)

Reply:
You are right, we need to document this. I'll follow-up on that.